src/infra/nodemailer/setup.js

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import * as dotenv from 'dotenv';
+import { generatingQRCode } from '../../utils/generateQRCode.js';
+
+dotenv.config();
+
+let transporter = nodemailer.createTransport({
```

Comment:
Como a `transporter` nunca muda, ela poderia ser inicializada como uma const.
```suggestion
const transporter = nodemailer.createTransport({
```

Reply:
Certo, fiz assim, pois foi sugestão da propia docs.